src/network/networkd-manager.c

```diff
@@ -1516,6 +1518,10 @@ static int manager_save(Manager *m) {
        if (!sip)
                 return -ENOMEM;
 
+       log = ordered_set_new(&string_hash_ops);
+       if (!log)
+               return -ENOMEM;
+
```

Comment:
Indentation is borked here, and for `sip` above. Why `log` — that's very misleading. Please call is something else.